atd-etl/cris_import/docker-compose.yml

```diff
@@ -1,5 +1,3 @@
-version: '3'
-
```

Comment:
Tag along change; `docker compose` not `docker-compose`, no longer need version designation. Same as many other PRs. LMK if you have any questions on this.